### ISSUE TITLE
(Release 3.2) Type explorer - click/navigation not working

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria-react-ui#268]
* egeria-base helm chart version 'egeria-base-3.2.0-prerelease.2'
* Includes egeria 3.2, egeria-react-ui 3.2
* Go to <psbaseurl>/org
* Go into type explorer
* Select mds1 (the only server deployed)

Several issues
* In the 'Entity Inheritance' view, clicking (single or double) on any type (label or dot) DOES NOT cause the type information to appear in the details pane. It remains blank 
* Whilst types can be loaded by selecting from the dropdown top right (and looked at in detail), in the 'neighbourhood' view, clicking on a type results in `Unknown type selection category in FocusContext undefined for type undefined`

**Comment by planetf1** [comment]
I have tested this & it is now fixed (including in 3.2) - thanks.
Closing

**Comment by bogdan-sava** [comment]
This was caused by a bug in d3 version.
The method needs the name of the type.

There are version where can be read from d.data.name and other d3 versions where is on d.currentTarget.textContent.
There are other versions where d.currentTarget is undefined.
On latest version of d3 (7.3.0) both can be used.

However I'm in the favor of consistence, because this change was done only on one place.


```
      nodeEnter.append("text")
               .attr("dy", "0.31em")
               .attr("x", 12)
               .attr("text-anchor", "start")
               .text(d => typesContext.isTypeDeprecated("Entity", d.data.name) ? "["+d.data.name+"]" : d.data.name )
               .on("click", d => { typeSelected("Entity", d.currentTarget.textContent); })
               .clone(true)
               .lower()
               .attr("stroke-linejoin", "round")
               .attr("stroke-width", 3)
               .attr("stroke", "white");
```


My suggestion is to update d3 to latest version and go back to d.data.name
